app/assets/stylesheets/provider/_links.scss

```diff
@@ -8,7 +8,7 @@ p a {
 }
 
 a {
-  text-decoration: none;
+  text-decoration: none !important; // Overrides Patternfly 4 links decoration introduced by @patternfly/react
```

Comment:
doesn't this override it everywhere? Links in other places should probably be underlined. But that might be already in place with more specific style rules in our css.

Reply:
I haven't looked very thoroughly, it might override some element when it shouldn't.

Reply:
for example ![Uploading Screenshot 2019-07-10 13.41.56.png…]() on dashboard should stay underlined so please check.

Reply:
I'd say a more specific override for vertical nav, horizontal dashboard nav & popup menus is safer

Reply:
What about all the links in the main content? Those shouldn't have an underline either.